R/pool.R

```diff
@@ -107,7 +107,7 @@ pool <- function(
 #' either `"rubin"`, `"jackknife"` or "`bootstrap"`.
 get_pool_components <- function(x) {
     switch(x,
-           "rubin" = c("est", "df", "se"),
+           "rubin" = c("est", "se", "df"),
```

Comment:
Any reason for the switch here ?

Reply:
No :( , I think I changed the order by mistake